src/promptflow-devkit/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -65,6 +65,29 @@ def list(
             message_generator=lambda x: f"Error parsing run {x.name!r}, skipped.",
         )
 
+    @monitor_operation(activity_name="pf.runs.search", activity_type=ActivityType.INTERNALCALL)
+    def search(
```

Comment:
Let's make this private now. 
Before we make this public:
- We need to go through some internal review to. 
- And we need to also make the pfazure has parity feature.

Reply:
Hello @wangchao1230  I thought that putting ```activity_type=ActivityType.INTERNALCALL``` was enough. Should I remove the @monitor_operation completely ?

Reply:
It will appear in reference doc if this is public fucnction: https://microsoft.github.io/promptflow/reference/python-library-reference/promptflow-devkit/promptflow.operations.html#promptflow.operations.RunOperations

It's just rename search() to _search()

Reply:
Done